Fix release build and make discovery period configurable

AndrejMitrovic opened the pull request:
Let's see if CI likes the first commit.

AndrejMitrovic committed:
Fix release build vanishing code

AndrejMitrovic committed:
Use simplified Point operators

The '+' operator was fixed previously.

linked0 reviewed (approved):
LGTM

hewison-chris reviewed (approved):
LGTM

kchulj reviewed (approved):
LGTM

Geod24 commented on source/agora/node/Ledger.d:
```diff
@@ -868,8 +868,8 @@ public class Ledger
             Point R = CR + challenge.toPoint();
-            sum_K = sum_K == Point.init ? K : (sum_K + K);
-            sum_R = sum_R == Point.init ? R : (sum_R + R);
+            sum_K = sum_K + K;
+            sum_R = sum_R + R;
```
I didn't know we fixed this in `crypto`, nice!

Geod24 commented:
This times out on `ValidatorCleanRestart`.

Geod24 commented on source/agora/node/Validator.d:
```diff
@@ -245,13 +245,9 @@ public class Validator : FullNode, API
-        });
+        void discover () { this.network.discover(this.required_peer_keys); }
+        discover();  // avoid delay
+        this.timers ~= this.taskman.setTimer(5.seconds, &discover, Periodic.Yes);
```
This interval should really be configurable

AndrejMitrovic replied:
Good point, let me do it now.

Geod24 commented:
Look at the integration test logs. First commit doesn't seem to do good there.

AndrejMitrovic commented:
Ah so it was there because of vibe.d.. then I'll have to nuke the commit.

AndrejMitrovic committed:
Manager: Make config protected

The only reason it was public was because
FullNode was accessing it. But FullNode
already has access to its configuration.

AndrejMitrovic committed:
Make network discovery period configurable

It was previously hardcoded to 5 seconds.

Geod24 reviewed (approved):
Can you rebase on v0.x.x ?

AndrejMitrovic commented:
Rebased.

Geod24 commented:
> Rebased.

CI disagrees
